src/redux/reducer.js

```diff
@@ -2,11 +2,13 @@ import { combineReducers } from 'redux';
 import articles from './reducers/articles';
 import authUser from './reducers/authUser';
 import common from './reducers/common';
-import { routerReducer } from 'react-router-redux';
+//import { routerReducer } from 'react-router-redux'; ==> OLD
```

Comment:
No need of comments here. Just remove it.

Reply:
Hi yeah maybe :) i do this long time ago when i learn React / Redux ;) Thix for the review